assets/src/editor/events/dates-and-times/editor-date/filter-bar/date-list-filter-bar.js

```diff
@@ -15,13 +15,13 @@ import { __ } from '@eventespresso/i18n';
  * @param {Function} onDisplayFilterChange
  * @return {Object} EditorDatesListView with added DateListFilters
  */
-class DatesListFilterBar extends Component {
+class DateListFilterBar extends Component {
 	static propTypes = {
 		showDates: PropTypes.string.isRequired,
-		sortDates: PropTypes.string.isRequired,
+		datesSortedBy: PropTypes.string.isRequired,
```

Comment:
`sortDates` seemed more imperative (i.e. I would expect it'd be a function for sorting dates).  So I thought doing this name change would make it a bit clearer that it refers to how dates are sorted.